nsot/models.py

```diff
@@ -925,6 +923,8 @@ def get_next_network(self, prefix_length, num=None, as_objects=True):
 
         log.info('>> WANTED = %s', wanted)
 
+        elapsed_time = time.time() - start_time
```

Comment:
Similar to above, was this meant to stick around? Sounds like this was part of the logging sequence meant for diagnosing performance.